Tools/LyTestTools/ly_test_tools/launchers/platforms/linux/launcher.py

```diff
@@ -228,40 +228,25 @@ def binary_path(self):
         return os.path.join(self.workspace.paths.build_directory(), "Editor")
 
 
-class LinuxGenericLauncher(LinuxLauncher):
```

Comment:
Do no other tests depend on the Generic variants now? Great to remove this, but only if it is no longer in use.

Reply:
no longer in use